src/main/java/net/tirasa/connid/bundles/zimbra/ZimbraConnector.java

```diff
@@ -317,6 +318,16 @@ public Uid create(ObjectClass objClass, Set<Attribute> attrs, OperationOptions o
                         sp().addAlias(account, toObjectName(objClass, alias));
                     }
                 }
+                Attribute distributionListAttr = AttributeUtil.find("__DISTRIBUTION_LISTS__", attrs);
```

Comment:
You should define `__DISTRIBUTION_LISTS__` as constant

Reply:
No, I see you already have it as a constant; then you should use that everywhere.